contracts/utils/contracts/test/TestLibBytes.sol

```diff
@@ -22,9 +22,20 @@ import "../src/LibBytes.sol";
 
 
 contract TestLibBytes {
-    
+
     using LibBytes for bytes;
 
+    /// @dev Pops the last byte off of a byte array by modifying its length.
+    /// @param b Byte array that will be modified.
+    /// @return The byte that was popped off.
+    function publicPopLastByteStateful(bytes memory b)
```

Comment:
What's the point of this? Relic from your `sendTransactionAsync()` issues?

Reply:
Yes, this is a relic. I'll remove it.

Reply:
Because of a bug in the Github UI, this is actually a reply to the above comment.

I think I can remove `RichErrors.sol` now that we are going to switch over to libraries entirely. Do you see anything wrong with that?

Reply:
Nope! Let's remove it if we can.